app/src/lib/editors/registry.ts

```diff
@@ -0,0 +1,75 @@
+// This module is a stripped back version of winreg:
+// https://github.com/fresc81/node-winreg
+//
+// I was seeing significant overhead when spawning the process to enumerate
+// the keys found by `reg.exe`, and rather than trying to fix and potentially
+// regress other parts I've extracted just the bit that I need to use.
+
+import { spawn } from 'child_process'
+
+type RegistryEntry = {
+  name: string
+  type: string
+  value: string
+}
+
+const regPath = 'C:\\Windows\\System32\\reg.exe'
+
+const ITEM_PATTERN = /^(.*)\s(REG_SZ|REG_MULTI_SZ|REG_EXPAND_SZ|REG_DWORD|REG_QWORD|REG_BINARY|REG_NONE)\s+([^\s].*)$/
+
+export function readKeys(key: string): Promise<ReadonlyArray<RegistryEntry>> {
+  return new Promise<ReadonlyArray<RegistryEntry>>((resolve, reject) => {
+    const proc = spawn(regPath, ['QUERY', key], {
+      cwd: undefined,
+      env: process.env,
```

Comment:
This was borrowed from `winreg`. I'm not sure it's essential anyway, but I'll confirm.